<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeChangeForm</class>
 <widget class="QDialog" name="VolumeChangeForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>81</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>261</width>
    <height>81</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>261</width>
    <height>81</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>261</width>
    <height>81</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>261</width>
    <height>81</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Volume Change</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/logo.png</normaloff>img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#5B5B5B;
}
QLineEdit{
border-radius:5px;padding:5px;background-color:#A3A1A1;color:#000
}
QPushButton{
background-color:#888787;
border-radius:5px
}
QPushButton:hover{
background-color:#B0B0B0;
}
QPushButton:pressed{
background-color:#A4A4A4;
}
QPushButton#NetflixButton{
background-color:#2E2E2E
}
QPushButton:hover#NetflixButton{
background-color:#000
}

</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="volumeUpButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>50</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/volume-up.png</normaloff>img/volume-up.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="volumeDownButton">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>50</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/volume-down.png</normaloff>img/volume-down.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVolNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Volume + or -</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
